src/test/java/teammates/test/cases/util/SanitizationHelperTest.java

```diff
@@ -95,6 +97,58 @@ private void sanitizeHtml_receivesSanitized_returnsUnchanged() {
         assertEquals(sanitized, SanitizationHelper.sanitizeForHtml(sanitized));
     }
 
+    @Test
+    public void testSanitizeForHtmlList() {
+        List<String> input = new ArrayList<>();
+        input.add("apple <");
+        input.add("banana ' dogs ");
+        input.add("rollercoasters & tycoons");
+        input.add("");
+        input.add(null);
+
+        List<String> expected = new ArrayList<>();
+        expected.add("apple &lt;");
+        expected.add("banana &#39; dogs ");
+        expected.add("rollercoasters &amp; tycoons");
+        expected.add("");
+        expected.add(null);
+
+        // standard case
+        assertEquals(expected, SanitizationHelper.sanitizeForHtml(input));
+
+        // empty input list
+        assertEquals(new ArrayList<String>(), SanitizationHelper.sanitizeForHtml(new ArrayList<>()));
+
+        // null input list
+        assertEquals(null, SanitizationHelper.sanitizeForHtml((List<String>) null));
```

Comment:
Use `assertNull`. Also, I don't think the comments add much value. Try to follow the style of the other test cases for consistency:
https://github.com/TEAMMATES/teammates/blob/9811580aad4bcfbe670a0ced33f9ff92057b116c/src/test/java/teammates/test/cases/util/SanitizationHelperTest.java#L29-L31